Dockerfile

```diff
@@ -5,9 +5,9 @@ RUN dnf install git xorg-x11-server-Xvfb procps-ng wget 'dnf-command(builddep)'
 RUN dnf builddep gtk4 -y
 
 # build gtk4 from the latest release
-ADD https://download.gnome.org/sources/gtk/4.2/gtk-4.2.1.tar.xz /tmp/gtk-4.2.1.tar.xz
-RUN tar -xf /tmp/gtk-4.2.1.tar.xz --directory /tmp
-WORKDIR /tmp/gtk-4.2.1
+ADD https://download.gnome.org/sources/gtk/4.3/gtk-4.3.2.tar.xz /tmp/gtk-4.3.2.tar.xz
+RUN tar -xf /tmp/gtk-4.3.2.tar.xz --directory /tmp
+WORKDIR /tmp/gtk-4.3.2
```

Comment:
Please do this in a separate PR so we can merge it already. Otherwise the CI will fail because of the outdated image

Reply:
The image should be ready once this job https://github.com/gtk-rs/gtk4-rs/runs/3172466469 is over